### ISSUE TITLE
tidy up pelican call

### ISSUE THREAD

**Issue by fangohr** [oscovida/oscovida#257]
We get a lot of error messages when running pelican:

```
cd tools/pelican && make html
117
make[1]: Entering directory '/home/runner/work/oscovida/oscovida/tools/pelican'
118
pelican /home/runner/work/oscovida/oscovida/tools/pelican/content -o /home/runner/work/oscovida/oscovida/tools/pelican/../wwwroot -s /home/runner/work/oscovida/oscovida/tools/pelican/pelicanconf.py 
119
WARNING: Watched path does not exist: /home/runner/work/oscovida/oscovida/tools/pelican/content/content/pages
120
WARNING: Watched path does not exist: /home/runner/work/oscovida/oscovida/tools/pelican/content/content/news
121
WARNING: Watched path does not exist: /home/runner/work/oscovida/oscovida/tools/pelican/content/content/ipynb
122
ERROR: Could not process ipynb/2020-are-summer-holidays-triggering-rising-cases.ipynb
123
  | basic
124
ERROR: Could not process ipynb/tutorial-overview-plot_time_step.ipynb
125
  | basic
126
ERROR: Could not process ipynb/2020-germany-reporting-delay-meldeverzug.ipynb
127
  | basic
128
ERROR: Could not process ipynb/tutorial-overview-plot_reproduction_number.ipynb
129
  | basic
130
ERROR: Could not process ipynb/tutorial-overview-plot_doubing_time.ipynb
131
  | Error processing f{filepath}: Could not find metadata in: .nbdata file or in the first cell of the notebook.If this notebook is used with liquid tags then you can safely ignore this error.
132
ERROR: Could not process ipynb/tutorial-overview-graphs.ipynb
133
  | basic
134
ERROR: Could not process ipynb/14-day-incidence-germany.ipynb
135
  | basic
136
ERROR: Could not process ipynb/tutorial-observables.ipynb
137
  | basic
138
ERROR: Could not process ipynb/tutorial-overview-plot_daily_change.ipynb
139
  | basic
140
ERROR: Could not process ipynb/2020-05-10-notebook-weekly-fluctuations-in-data-from-germany.ipynb
141
  | basic
142
ERROR: Could not process ipynb/r-value.ipynb
143
  | basic
144
ERROR: Could not process ipynb/tutorial-accessing-cases-and-deaths.ipynb
145
  | basic
146
ERROR: Could not process ipynb/14-day-incidence.ipynb
147
  | basic
148
ERROR: Could not process ipynb/tutorial-accessing-population-data.ipynb
149
  | basic
150
ERROR: Could not process ipynb/compare-rki-and-johns-hopkins-data.ipynb
151
  | basic
152
ERROR: Could not process ipynb/tutorial-exponential-growth-example.ipynb
153
  | basic
154
ERROR: Could not process ipynb/2020-04-30-bbc-germany-increasing.ipynb
155
  | basic
156
ERROR: Could not process ipynb/tutorial-installation.ipynb
157
  | basic
158
ERROR: Could not process ipynb/2020-thanksgiving-us.ipynb
159
  | basic
160
WARNING: AUTHOR_SAVE_AS is set to False
161
Done: Processed 17 articles, 1 draft, 1 page, 0 hidden pages and 1 draft page in 3.45 seconds.
162
make[1]: Leaving directory '/home/runner/work/oscovida/oscovida/tools/pelican'
```

This doesn't seem to affect the outcome. We should check that this is the case, and if so get rid of these (unnerving) errors.

**Comment by kirienko** [comment]
Is is from the CI or form your local run?

**Comment by kirienko** [comment]
It might be due to `nbconvert` again: https://github.com/jupyter/nbconvert/issues/1369

**Comment by kirienko** [comment]
more details here: https://github.com/danielfrg/pelican-jupyter/issues/126

**Comment by kirienko** [comment]
Well, maybe we need to go from `pelican` to something more... well... maintainable. Like `mkdocs` or `jupyterbook`. But that's much more work of course.

**Comment by fangohr** [comment]
This is fixed with PR #280 and #278
